test.js

```diff
@@ -1,6 +1,8 @@
 // @flow
-const convert = require('./index');
-const { resolveToLast } = require('./utils');
+const k2s = require('./dist');
```

Comment:
Why are you pulling this from `dist`?

Reply:
Bad reasons. Raised an issue for me to fix.